libs/architectures/aframe/architectures/resnet.py

```diff
@@ -12,6 +12,11 @@
 
 
 class ChannelNorm(torch.nn.Module):
+    """
+    Custom implementation of ChannelNorm which is faster than the
+    out-of-the-box PyTorch version.
```

Comment:
Also now that I'm looking at this I don't know why I call it `ChannelNorm` rather than `GroupNorm`

Reply:
You want me to change it? And then have `get_norm_layer` return this instead?

Reply:
I think you'll still need the wrapper module below that creates a class whose only argument is `num_channels` (since that's all that gets passed at each layer, and makes things more robust to different norm layers), but yeah I would maybe call this `GroupNorm` and then just do something lazy like `_GroupNorm` or `NormLayer` for the subclass defined in the wrapper.